tsqc/__main__.py

```diff
@@ -116,6 +117,7 @@ def setup_logging(log_level, no_log_filter):
 
 @click.command()
 @click.argument("path", type=click.Path(exists=True, dir_okay=False))
+@click.option("--annotate-genes", type=click.Path(exists=True, dir_okay=False))
```

Comment:
Maybe "annotations-file`` would be more intuitive? You can then use this name throughout